src/main/java/nextstep/ladder/domain/ladder/Direction.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.ladder.domain.ladder;
+
+import nextstep.ladder.domain.exceptions.InvalidDirectionParameterException;
+
+import java.util.Objects;
+
+public class Direction {
+    private boolean currentPointStatus;
+    private boolean nextPointStatus;
+
+    public Direction(boolean currentPointStatus, boolean nextPointStatus) {
+        validate(currentPointStatus, nextPointStatus);
+        this.currentPointStatus = currentPointStatus;
+        this.nextPointStatus = nextPointStatus;
+    }
+
+    public boolean getCurrentStatus() {
+        return this.currentPointStatus;
+    }
+
+    public boolean getNextStatus() {
+        return this.nextPointStatus;
+    }
+
+    public Direction next(PointAddStrategy pointAddStrategy) {
+        boolean nextDirectionCurrentStatus = this.nextPointStatus;
+        return new Direction(
+                nextDirectionCurrentStatus, pointAddStrategy.confirmPointLocation(!nextDirectionCurrentStatus));
```

Comment:
```suggestion
        return new Direction(
                 this.nextPointStatus, pointAddStrategy.confirmPointLocation(!this.nextPointStatus));
```
개인 코딩 스타일이지만..바로 이렇게 써도 될 것 같습니다:)